layouts/index.html

```diff
@@ -13,8 +13,8 @@
     id="main">
   <div class="usa-grid section-bg-parent">
     <div class="usa-width-whole billboard  section-bg">
-      <h1 class="billboard-display_text">The Government Innovation Platform</h1>
-      <h2 class="billboard-second_text">A platform by government developers, for government developers.</h1>
+      <h1 class="billboard-display_text">A Platform as a Service for government teams</h1>
```

Comment:
I agree with cutting line 16, though we should perhaps replace it? For now, we could simply replace it with "Cloud.gov." Otherwise, we don't really emphasize the title of this product on the overview page.

RE: line 17 (PaaS), I hear Bret's concern, though my reaction is that PaaS is a very googleable term, whereas CaaS is less so, or when googling this term, the results present a less standardized concept.

Perhaps we're not at the final point but I still find these edits to be improvements so I'd like to merge with the intention that we'll keep tweaking and improving. Good @mogul?